frontends/main/src/page-components/AiChat/AskTimDrawerButton.tsx

```diff
@@ -1,44 +1,29 @@
 import React from "react"
-import { Typography, styled } from "ol-components"
-import { ButtonLink } from "@mitodl/smoot-design"
+import { Typography, styled, LinkAdapter } from "ol-components"
 import { RiSparkling2Line } from "@remixicon/react"
 import AiRecommendationBotDrawer from "./AiRecommendationBotDrawer"
 import { RECOMMENDER_QUERY_PARAM } from "@/common/urls"
 
-const StyledButton = styled(ButtonLink)(({ theme }) => ({
+const StyledButton = styled(LinkAdapter)(({ theme }) => ({
```

Comment:
We shouldn't use ButtonLink here. We don't want any of its styling.

As far as I can tell, we only used it because we wanted the `shallow` prop, which comes from LinkAdapter.